tsl/src/nodes/decompress_chunk/compressed_batch.c

```diff
@@ -320,87 +437,15 @@ compressed_batch_set_compressed_tuple(DecompressChunkState *chunk_state,
 			case COMPRESSED_COLUMN:
 			{
 				Assert(i < chunk_state->num_compressed_columns);
+				/*
```

Comment:
Could you add this information also to the pr/commit message?

Reply:
I updated the messages.